pkg/az/ops.go

```diff
@@ -43,27 +43,20 @@ func errFailedToGetEnvironment(err error) {
 		Msg("Failed to get environment.")
 }
 
-func errFailedToGetAuthrorizerFromCli(err error) {
+func errFailedToGetAuthorizerFromCli(err error) {
```

Comment:
This will be removed?

Reply:
Nope it's just typo fixed in: Auth**r**orizer